avro-serializer/src/main/java/io/confluent/kafka/formatter/AvroMessageReader.java

```diff
@@ -204,4 +208,8 @@ private Object jsonToAvro(String jsonString, Schema schema) {
   public void close() {
     // nothing to do
   }
+
+  void setEnableAutoSchemaRegistration(boolean flag) {
```

Comment:
There is already a constructor specifically for testing - maybe reuse that instead of adding this method.